Dockerfile

```diff
@@ -52,5 +52,15 @@ RUN mkdir /html \
 
 # Copy the HTML and serve it via nginx
 FROM nginx:alpine
+
+ARG BUILD_DATE=""
+ARG BUILD_VERSION="dev"
```

Comment:
how does this get overwritten by the actual tag?

Reply:
`        docker build -t \
          ${{ steps.vars.outputs.name }}:${{ steps.vars.outputs.tag }} \
          --build-arg BUILD_DATE=${{ steps.vars.outputs.date }} \
          --build-arg BUILD_VERSION=${{ steps.vars.outputs.version }} \
          .`